mysql-test/suite/tokudb/t/locking-read-repeatable-read-1.test

```diff
@@ -0,0 +1,27 @@
+source include/have_tokudb.inc;
+source include/count_sessions.inc;
+
+--disable_warnings
+drop table if exists t;
+--enable_warnings
```

Comment:
This is an obsolete practice, no need to 'cleanup' before you begin your test. Please remove. Sorry I missed it the first time through. Only need to change, re-record, and ensure that just this test passes jenkins, not the full suite (use mtr args of '--suite=tokudb locking-read-repeatable-read-1').

Reply:
Ah? I thought it was a good practice to ensure everything is fresh. I will remove but why is it an obsolete practice? :)

Reply:
The mtr runner validates the environment when the test is completed to ensure that the test has cleaned some things up after itself (databases and tables removed, etc...). It is not totally complete but getting better at this over time. This makes it unnecessary for a test to pre-clean.